### PR TITLE
added morpheus as character

### PR DESCRIPTION
**Pull request by Naman73598**
Updated the character for Morpheus.
#hacktoberfest2023

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,13 @@ <h4>Name: Trinity</h4>
           <p>Quotes: "The Matrix isn't real." </p>
         </div>
 
+        <div class="character">
+          <h4>Name: Morpheus</h4>
+          <img src="/img/characters/Morpheus.jpg" alt="Morpheus" />
```
Comment (roannav) [inline index.html]: The photo file is named "morpheus.jpg", but the `` tag is "Morpheus.jpg".  Please correct it to be lowercase in the `` tag.

**Review by roannav — CHANGES_REQUESTED** [review]
The photo file is named "morpheus.jpg".  However, in index.html, the `` tag is "Morpheus.jpg".  Please correct it to be lowercase in the `` tag.

Then git add, git commit, and git push again.

**Comment by Naman73598** [comment]
Changes made . Please check @roannav

**Review by roannav — APPROVED** [review]

**Comment by roannav** [comment]
@Naman73598  Thank you very much for your contribution! 🏆

**Comment by roannav** [comment]
@all-contributors please add @Naman73598 for code and content

**Pull request merged**